readme.md

```diff
@@ -1,3 +1,186 @@
+```
+
+Once Pine Core is set up, you can proceed with installing and using Pine Icons either through npm (as shown in Quick Start) or via CDN.
+
+## Installation & Setup
+
+### Using npm
+
+```bash
+npm install @pine-ds/icons
+```
+
+### Using yarn
+
+```bash
+yarn add @pine-ds/icons
+```
+
+### Import and Registration
+
+```javascript
+// Import in your main application file
+import '@pine-ds/icons';
+// The web components will be automatically registered
+```
+
+## Usage
+
+### Basic Usage
+
+```html
+<pds-icon name="icon-name"></pds-icon>
+```
+
+### Customization Options
+
+```html
+<!-- Custom size -->
+<pds-icon name="icon-name" size="24px"></pds-icon>
+
+<!-- Custom color -->
+<pds-icon name="icon-name" color="#663399"></pds-icon>
+```
+
+### Available Properties
+
+| Property | Type   | Default         | Description                                                         |
+|----------|--------|-----------------|---------------------------------------------------------------------|
+| name     | string | **required**    | The name of the icon to display                                     |
+| size     | number | `regular`            | Size of the icon in pixels                                          |
+| color    | string | | Color of the icon (supports hex, rgb, Pine token, or CSS color names) |
+
+## Contributing
+
+We welcome contributions! Please follow these guidelines when contributing to the project.
+
+### Development Workflow
+
+1. **Fork the repository.**
+2. **Clone:**
+   ```bash
+   git clone https://github.com/YOUR_USERNAME/pine-icons.git
+   cd pine-icons
+   ```
+3. **Create your branch:**
+   ```bash
+   git checkout -b style/your-style-update
+   ```
+4. **Install dependencies:**
+   ```bash
+   npm ci
+   ```
+5. **Develop and test your changes:**
+   ```bash
+   npm start
+   npm test
+   ```
+6. **Commit using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/):**
+   ```bash
+   git commit -m 'style(header): add new header styles'
+   ```
+7. **Push your branch:**
+   ```bash
+   git push origin style/your-style-update
+   ```
+8. **Open a Pull Request.**
+
+### Development Prerequisites
+
+- **Node.js:** v22.7.0 or higher
+- **npm:** (comes with Node.js)
+- **Git**
+
+### Local Development
+
+1. **Start the development server:**
+   ```bash
+   npm start
+   ```
+   The server runs on **port 7200** and provides a live preview.
+2. **Run tests:** Ensure your changes pass all tests.
+
+### Available Commands
+
+| Command                                   | Description                                      |
+|-------------------------------------------|--------------------------------------------------|
+| `npm start`                               | Start development server with live preview       |
+| `npx nx run @pine-ds/icons:build`           | Build the icon library for production            |
+| `npx nx run @pine-ds/icons:update`          | Update icons from Figma source                   |
```

Comment:
We probably don't want to expose this, although it's in the package.json. To run this, they must have the proper Figma Credentials in their local environment.